### PR TITLE
More informative error message for image decoding failure.

### PR DESCRIPTION
**Pull request by amirh**
fixes https://github.com/flutter/flutter/issues/13147

### CODE REVIEW

File: lib/ui/painting/codec.cc [hunk 1]
```diff
@@ -75,7 +75,8 @@ fxl::RefPtr<Codec> InitCodec(sk_sp<SkData> buffer, size_t trace_id) {
 
   std::unique_ptr<SkCodec> skCodec = SkCodec::MakeFromData(buffer);
   if (!skCodec) {
-    FXL_LOG(ERROR) << "SkCodec::MakeFromData failed";
+    FXL_LOG(ERROR) << "Image data is either invalid, or not a supported "
+                      "format, decoding failed.";
```
Comment (yjbanov) [inline lib/ui/painting/codec.cc]: nits:
- "or it is encoded using an unsupported format"
- "Decoding image failed" would read better at the beginning as the user is more likely to be interested in what happened before the explanation of why it happened.

**Comment by yjbanov** [comment]
lgtm

File: lib/ui/painting/codec.cc (hunk 1 above)
Comment (amirh) [reply lib/ui/painting/codec.cc]: PTAL

**Comment by Hixie** [comment]
You're missing a period at the end of the sentence, but otherwise LGTM. :-)

**Pull request merged**